src/components/Bookings.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from "react";
+import Search from "./Search.js";
+import SearchResults from "./SearchResults.js";
+
+const Bookings = () => {
+  const [bookings, setBookings] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
```

Comment:
This is a really nice way of dealing with errors and delayed data🏅 Conditional rendering works very well too!